divide_and_conquer/closest_pair_of_points.py

```diff
@@ -1,27 +1,27 @@
 """
-The algorithm finds distance btw closest pair of points in the given n points.
+The algorithm finds distance between closest pair of points 
+in the given n points.
 Approach used -> Divide and conquer 
-The points are sorted based on Xco-ords 
+The points are sorted based on Xco-ords and 
+then based on Yco-ords separately.
 & by applying divide and conquer approach, 
 minimum distance is obtained recursively.
 
->> closest points lie on different sides of partition
+>> closest points can lie on different sides of partition.
```

Comment:
It's better to make the first character of each sentence upper case. Please check other sentences too.